docs/index.rst

```diff
@@ -1,51 +1,211 @@
 .. _index:
 
-=====================
+#####################
 CrateDB Python Client
-=====================
+#####################
 
-A Python client library for `CrateDB`_.
+.. rubric:: Table of contents
 
-This client library implements the `Python Database API Specification v2.0`_
-(PEP 249), which defines a common interface for accessing databases in Python.
+.. contents::
+    :local:
+    :depth: 1
 
-It also includes the :ref:`CrateDB dialect <using-sqlalchemy>` for `SQLAlchemy`_.
 
-.. NOTE::
+************
+Introduction
+************
 
-    This is a basic CrateDB driver reference.
+The Python client library for `CrateDB`_ implements the Python Database API
+Specification v2.0 (`PEP 249`_), and also includes the :ref:`CrateDB dialect
+<using-sqlalchemy>` for `SQLAlchemy`_.
 
-    Check out the `sample application`_ (and the corresponding `sample
-    application documentation`_) for a practical demonstration of this driver
-    in use.
+The Python driver can be used to connect to both `CrateDB OSS`_ and `CrateDB
```

Comment:
I don't think we refer to CrateDB as CrateDB OSS anywhere, do we?
Also I am not sure if we really should differentiate. CrateDB Cloud is mostly a CrateDB instance with user management and TLS enabled and therefore a "correct" setup CrateDB "OSS" instance would behave similar to a CrateDB Cloud instance

Reply:
I also was not sure. I will remove that term again in order not to cause any confusion.

Reply:
Fixed with 8235a75be0.